packages/scss/src/mixins/_utilities.scss

```diff
@@ -20,8 +20,12 @@
 
 @mixin listReset($suffix: '') {
 	list-style-type: none #{$suffix};
-	margin: 0 #{$suffix};
 	padding: 0 #{$suffix};
+	
+	&,
+	> * {
```

Comment:
I can seesome potentials side effects to add `> *` in this reset. I think it would be safer to juste create a specific reset for dl dt dd lists

Reply:
You're right! At least for the `div` nested in the `dl`.

How about a `&, dd { margin: 0; }` ?

Reply:
There are 2 way to do it :
- As you suggest, just add `&, dt, dd { margin: 0; }` in the reset
- Or create a new `u-definitionListReset` with
```
@include listReset;
&, > * { margin: 0; }
```
i lean toward the second option, since list and definition list are slightly different DOM wise.